projects/05-fleet-management-api/project.yml

```
@@ -8,3 +8,7 @@ learningObjectives:
   - db
   - postgres/data-types
   - postgres/indexes
+  - scm/git
+  - scm/github/setup
+  - scm/github/gh-pages
+  - scm/github/collaboration
```

Aqui y por el otro creo seria:

```suggestion
  - scm/git
  - scm/github/setup
  - scm/github/collaboration
  - scm/github/project-management
```
Omito `gh-pages` porque solo desplegamos FM API, y seria con vercel (no gh-pages)

@vivieall @ssinuco 
Me pregunto si seguimos agregando los OAs basicos de
`scm/git/setup`
y
`scm/github/setup` 
para proyectos de rutas.